internal/services/cockpit/alert_manager.go

```diff
@@ -0,0 +1,256 @@
+package cockpit
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/cockpit/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/locality/regional"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/services/account"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/types"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/verify"
+)
+
+func ResourceCockpitAlertManager() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: ResourceCockpitAlertManagerCreate,
+		ReadContext:   ResourceCockpitAlertManagerRead,
+		UpdateContext: ResourceCockpitAlertManagerUpdate,
+		DeleteContext: ResourceCockpitAlertManagerDelete,
+		Timeouts: &schema.ResourceTimeout{
+			Create:  schema.DefaultTimeout(DefaultCockpitTimeout),
+			Read:    schema.DefaultTimeout(DefaultCockpitTimeout),
+			Delete:  schema.DefaultTimeout(DefaultCockpitTimeout),
+			Default: schema.DefaultTimeout(DefaultCockpitTimeout),
+		},
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"project_id": account.ProjectIDSchema(),
+			"enable_managed_alerts": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"emails": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString, ValidateFunc: verify.IsEmail()},
+				Optional:    true,
+				Description: "A list of email addresses for the alert receivers",
+			},
+			"region": regional.Schema(),
+			"alert_manager_url": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Alert manager URL",
+			},
+		},
+	}
+}
+
+func ResourceCockpitAlertManagerCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	api, region, err := cockpitAPIWithRegion(d, meta)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	projectID := d.Get("project_id").(string)
+	emails := d.Get("emails").([]interface{})
+
+	_, err = api.EnableAlertManager(&cockpit.RegionalAPIEnableAlertManagerRequest{
+		Region:    region,
+		ProjectID: projectID,
+	})
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	_, err = api.EnableManagedAlerts(&cockpit.RegionalAPIEnableManagedAlertsRequest{
+		Region:    region,
+		ProjectID: projectID,
+	})
+	if err != nil {
+		return diag.FromErr(err)
+	}
```

Comment:
Do we want to always enable managed alerts by default ? If not I we should enable them only when the user sets the field to true

```suggestion
	_, managedAlertsExists := d.GetOk("enable_managed_alerts")
	if managedAlertsExists {
		_, err := cockpitApi.EnableManagedAlerts(&cockpit.RegionalAPIEnableManagedAlertsRequest{
			Region:    region,
			ProjectID: d.Get("project_id").(string),
		})
		if err != nil {
			return diag.FromErr(err)
		}
	}
```